job-manager/example/src/test/java/uk/gov/moj/cpp/jobmanager/example/BakeryServiceIT.java

```diff
@@ -125,10 +134,18 @@ public void setup() throws Exception {
     }
 
     public void initJobStoreDatabase() throws Exception {
+        final ClassLoaderResourceAccessor resourceAccessor = new ClassLoaderResourceAccessor();
+        final JdbcConnection connection = new JdbcConnection(dataSource.getConnection());
+
+        new Liquibase(
+                "liquibase/jobstore-db-changelog.xml",
```

Comment:
For this same file , we are calling dropAll() then later on line 146 we are doing similar with call to update()

Do we need both?